app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -1288,9 +1296,34 @@ public void onReceivedTitle(@NonNull Tab tab, String title) {
         public void onReceivedIcon(@NonNull Tab tab, Bitmap icon) {
         }
 
+        private void switchToTab(Tab targetTab) {
+            // ensure it does not have attach to parent earlier.
+            targetTab.detach();
+
+            @Nullable final View outView = findExistingTabView(webViewSlot);
+            webViewSlot.removeView(outView);
+
+            final View inView = targetTab.getTabView().getView();
+            webViewSlot.addView(inView);
+
+            startTransitionAnimation(null, inView, null);
+        }
+
+        private void onForegroundTabHoist(Tab tab) {
+            geolocationOrigin = "";
+            geolocationCallback = null;
+
+            dismissGeoDialog();
+
+            updateURL(tab.getUrl());
+
+            int identity = tab.getSecurityState() == SiteIdentity.SECURE ? SITE_LOCK : SITE_GLOBE;
```

Comment:
please `(tab.getSecurityState() ==  SiteIdentity.SECURE) ? SITE_LOCK : SITE_GLOBE;`

Reply:
updated and force-pushed